modules/accounts/calendar.nix

```diff
@@ -0,0 +1,111 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.accounts.calendar;
+
+  calendarOpts = { name, config, ... }: {
+    options = {
+      name = mkOption {
+        type = types.str;
+        readOnly = true;
+        description = ''
+          Unique identifier of the calendar. This is set to the
+          attribute name of the calendar configuration.
+        '';
+      };
+
+      path = mkOption {
```

Comment:
This should go into some form of submodule corresponding to the `maildir` submodule in the email account module. Not sure whether there is any widely supported local storage of calendars and contacts. I guess single vcf, vcs, or ics containing multiple entries is common? Or basic directory structures of single files?

Could then have `vcfFile` and `vcfDirectory` as sub-modules. Should be `null`-able for people not wanting to sync locally.